infrastructure/docker-registry/harbor-values.yaml

```diff
@@ -0,0 +1,921 @@
+
+portal:
+  image:
+    repository: goharbor/harbor-portal
+    tag: v2.4.0
+  # set the service account to be used, default if left empty
+  serviceAccountName: ""
+  # mount the service account token
+  automountServiceAccountToken: false
+  replicas: 3
+  resources:
+    requests:
+      memory: 100Mi
+      cpu: 10m
+    limits:
+      memory: 150Mi
+      cpu: 20m
+  nodeSelector: {}
+  tolerations: []
+  affinity: 
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+            - key: component
+              operator: In
+              values:
+              - portal
+        topologyKey: kubernetes.io/hostname
+  ## Additional deployment annotations
+  podAnnotations: 
+  ## The priority class to run the pod as
+  priorityClassName:
+
+core:
+  image:
+    repository: goharbor/harbor-core
+    tag: v2.4.0
+  # set the service account to be used, default if left empty
+  serviceAccountName: ""
+  # mount the service account token
+  automountServiceAccountToken: false
+  replicas: 3
+  ## Startup probe values
+  startupProbe:
+    enabled: true
+    initialDelaySeconds: 10
+  resources:
+    requests:
+      memory: 150Mi
+      cpu: 50m
+    limits:
+      memory: 300Mi
+      cpu: 100m
+  nodeSelector: {}
+  tolerations: []
+  affinity: 
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+            - key: component
+              operator: In
+              values:
+              - core
+        topologyKey: kubernetes.io/hostname
+  ## Additional deployment annotations
+  podAnnotations: {}
+  # Secret is used when core server communicates with other components.
+  # If a secret key is not specified, Helm will generate one.
+  # Must be a string of 16 chars.
+  secret: ""
+  # Fill the name of a kubernetes secret if you want to use your own
+  # TLS certificate and private key for token encryption/decryption.
+  # The secret must contain keys named:
+  # "tls.crt" - the certificate
+  # "tls.key" - the private key
+  # The default key pair will be used if it isn't set
+  secretName: ""
+  # The XSRF key. Will be generated automatically if it isn't specified
+  xsrfKey: ""
+  ## The priority class to run the pod as
+  priorityClassName:
+
+jobservice:
+  image:
+    repository: goharbor/harbor-jobservice
+    tag: v2.4.0
+  replicas: 1
+  # set the service account to be used, default if left empty
+  serviceAccountName: ""
+  # mount the service account token
+  automountServiceAccountToken: false
+  maxJobWorkers: 10
+  # The logger for jobs: "file", "database" or "stdout"
+  jobLoggers:
+    - file
+    # - database
+    # - stdout
+
+  resources:
+    requests:
+      memory: 100Mi
+      cpu: 30m
+    limits:
+      memory: 200Mi
+      cpu: 60m
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  nodeAntiAffinityPreset: soft
+  podAntiAffinityPreset: soft
```

Comment:
These do not seem to me to be present in the original chart, and I doubt they would work.
```suggestion
```